### PR TITLE
@86epycczy #Enhance Alert component expiration

### PR DESCRIPTION
**Pull request by nasolgabriel**
enhance alert for expired product

### CODE REVIEW

File: packages/core-library/components/Alert/GenericAlert/Alert.tsx
Comment (DropitupQwerty) [inline packages/core-library/components/Alert/GenericAlert/Alert.tsx:1]: Did you create this file or this file is already existing?

Comment (DropitupQwerty) [reply packages/core-library/components/Alert/GenericAlert/Alert.tsx:1]: If this file was already existing and implemented on other products. Consult with other leads or other team on how they use this component so we can avoid affecting the other products

Comment (nasolgabriel) [reply packages/core-library/components/Alert/GenericAlert/Alert.tsx:1]: > Did you create this file or this file is already existing?
> If this file was already existing and implemented on other products. Consult with other leads or other team on how they use this component so we can avoid affecting the other products

the file already existed in genericAlert folder, will improve the code and consult leads and other team members

**Review by JMSevilla — APPROVED** [review]

**Review by be3jaay — APPROVED** [review]

File: packages/core-library/tests/components/Alerts/Alert.test.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
```
Comment (JezChristian) [inline packages/core-library/tests/components/Alerts/Alert.test.tsx:2]: We have our own render, and screen under ../../common. Thanks!

**Review by DropitupQwerty — APPROVED** [review]

**Review by acostales08 — APPROVED** [review]

File: packages/core-library/tests/components/Alerts/Alert.test.tsx (hunk 1 above)
Comment (nasolgabriel) [reply packages/core-library/tests/components/Alerts/Alert.test.tsx:2]: > We have our own render, and screen under ../../common. Thanks!
will edit the code immediately and use our own render. Thank you!

**Pull request merged**